### PR TITLE
feat(react-kit): the ability to disable the min and max buttons separ…

### PR DESCRIPTION
**Pull request by dramoturg**

### CODE REVIEW

**Comment by dramoturg** [comment]
> Looks like the state of the buttons should be controlled by NumericStepper itself, not from outside



> Looks like the state of the buttons should be controlled by NumericStepper itself, not from outside

add state to numeric stepper?

**Comment by raveclassic** [comment]
@dramoturg What kind of state? The state of the buttons is derived from value/min/max props

**Comment by sutarmin** [comment]
@raveclassic I'm not sure that your idea will fit all cases. The logic of disabling might be tricky when we deal with float point numbers

**Comment by raveclassic** [comment]
@sutarmin Well NumericStepper is already operates on the actual value adding/subtracting the step to the value. I don't see any problems with floats if you provide optional `precision` argument or smth similar.

**Comment by sutarmin** [comment]
@raveclassic, agreed.

Then it looks like we can incapsulate this logic inside component without changing its interface. We should just have something like `isDisabled={value + step > max}`

**Comment by raveclassic** [comment]
@sutarmin exactly

**Comment by raveclassic** [comment]
Even more, precision can be derived from the step

File: packages/react-kit/src/components/SteppableInput/SteppableInput.tsx [hunk 1]
```diff
@@ -21,6 +21,8 @@ export type TPickedInputProps = Pick<
 
 export type TFullSteppableInputProps = TPickedInputProps & {
 	isDisabled?: TInputProps['isDisabled'];
+	isDisabledMinButton?: boolean;
+	isDisabledMaxButton?: boolean;
```
Comment (sutarmin) [inline packages/react-kit/src/components/SteppableInput/SteppableInput.tsx]: It is not Max and Min, but Increment and Decrement in context of SteppableInput. And also I think from the perspective of the English language it is more correct `isIncrementButtonDisabled` than `isDisabledIncrementButton`

**Pull request merged**